src/dialog/dialog.component.ts

```diff
@@ -137,7 +137,11 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 	 */
 	ngAfterViewInit() {
 		const dialogElement = this.dialog.nativeElement;
-		dialogElement.classList = `${dialogElement.classList} ${this.dialogConfig.wrapperClass}`;
+		if (this.dialogConfig.wrapperClass) {
```

Comment:
Maybe add a comment to blame IE11 for the complication?

Reply:
Is github not showing the comment or you think it's no longer necessary?

Reply:
I actually totally forgot to add a comment 😅